theme/static/js/hs_file_browser.js

```diff
@@ -885,6 +886,36 @@ function bindFileBrowserItemEvents() {
                                 break;
                             }
                         }
+                        if (extensionApp) {
```

Comment:
I think we can improve these nested if blocks
L879 `if (fileSelected) {`
...
L889 `if (extensionApp) {`
...
}
}
L921 `if (fileSelected) {`
L922 `if (!extensionApp) {`
...

I think it would be clearer if we combine the two `if (fileSelected) {` blocks into a single block, then change `if (!extensionApp) {` -> `else {`

Reply:
Combined it to a single `if (fileSelected)` block.